examples/options/web/src/ui/theme.css

```
@@ -0,0 +1,504 @@
+  --ark-violet: #9b7bff;
+
+  --ark-green: #2bd576;
+  --ark-red: #ff5c5c;
+  --ark-amber: #ffc24b;
+
+  --ark-radius: 14px;
+  --ark-radius-sm: 9px;
+  --ark-shadow: 0 8px 30px rgba(0, 0, 0, 0.45);
+  --ark-glow: 0 0 0 1px rgba(200, 255, 61, 0.25), 0 0 24px rgba(200, 255, 61, 0.12);
+
+  --ark-mono: ui-monospace, "SF Mono", "JetBrains Mono", "Fira Code", Menlo, monospace;
```

Stylelint (17.13.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)